packages/components/src/color-picker/README.md

```diff
@@ -31,13 +31,13 @@ The current color value to display in the picker. Must be a hex or hex8 string.
 
 ### `onChange`: `(hex8Color: string) => void`
 
-Fired when the color changes. Always passes a hex8 color string.
+Fired when the color changes. Always passes a hex or hex8 color string.
```

Comment:
`onChange` previously always passed a hex8 string, but since #35562, `Colord.toHex()` will return a hex or a hex8 (only a hex8 if the color's alpha is < 1)

Reply:
Nice catch, thank you!